src/components/SectionCard.js

```diff
@@ -0,0 +1,17 @@
+import CircleGauge from './CircleGauge';
+
+function SectionCard({ section, percentage, subcategories, color }) {
+  return (
+    <div className="bg-white shadow-widgit flex flex-nowrap justify-around">
+      <CircleGauge className="h-24 w-24" percentage={percentage} color={color} />
+      <div className="grid place-items-center">
+        <div className="flex flex-col items-center">
+          <p className="font-sans font-bold text-{32px}">{section}</p>
+          <p className="text-xs text-gray-400">{subcategories} Subcategories</p>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default SectionCard;
```

Comment:
A few things: 
1. The cards should be able to handle click events, as described in the task. I'd recommend passing eventhandler functions in as a prop. As for styling the selected card, we can use a `className` prop.
2. Additionally, using a `className`prop lets us remove the OverallCard component; any of the col-span magic can also be passed in through that prop. The differences in subheaders – patients vs subcategories – can be resolved by having the entire subheader string be passed in instead of just the category count vs patient number.